src/main/java/org/sagebionetworks/web/client/presenter/EntityPresenter.java

```diff
@@ -268,6 +273,9 @@ public Widget asWidget() {
 
 	@EventHandler
 	public void onEntityUpdatedEvent(EntityUpdatedEvent event) {
+		SynapseReactClientQueryKey queryKey = SynapseReactClientQueryKey.create("entity", event.getEntityId());
+		// The query key is an array where the first element is the object we constructed
+		queryClient.invalidateQueries(new Object[]{queryKey});

```

Comment:
Invalidate the react-query client for the updated entity, or all entities if the event's entity ID is null